addons/binding/org.openhab.binding.kodi/README.md

```diff
@@ -105,16 +105,16 @@ sitemap demo label="myKodi"
     Frame label="myKodi" {
         Switch    item=myKodi_mute
         Slider    item=myKodi_volume
-        Selection item=myKodi_control mappings=[PLAY='Play', PAUSE='Pause', NEXT='Next', PREVIOUSE'Previous', FFWARD='Fastforward', REWIND='Rewind']
+        Selection item=myKodi_control mappings=[PLAY='Play', PAUSE='Pause', NEXT='Next', PREVIOUSE'Previous', FASTFORWARD='Fastforward', REWIND='Rewind']
```

Comment:
Please add an equal sign between `PREVIOUSE` and `'Previous'` in the mapping array. It is missing. Additionally change the spelling of `PREVIOUSE` to `PREVIOUS`. That would be nice. Thanks.